packages/quantic/force-app/main/default/lwc/quanticResultQuickview/quanticResultQuickview.js

```diff
@@ -91,6 +91,17 @@ export default class QuanticResultQuickview extends LightningElement {
       // eslint-disable-next-line @lwc/lwc/no-inner-html
       this.contentContainer.innerHTML = this.state.content;
     }
+    const slots = this.template.querySelectorAll('slot');
+    slots.forEach((slot) => {
```

Comment:
here as well. A named function would make it clearer what is happening with this manipulation.